src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -93,6 +95,12 @@ private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
     this.processRunner = new DefaultProcessRunner(async, stdOutLineListeners, stdErrLineListeners,
         exitListeners, startListeners);
 
+    // Populate jar locations.
```

Comment:
something I just discovered: not all google cloud SDK installs have this. The app-engine-java component needs to be explicitly installed. 

Not for this PR, but we need to think about what happens if these paths aren't present.